Makefile

```diff
@@ -74,7 +74,7 @@ fmt:
 .PHONY: test
 # we say code is not worth testing unless it's formatted
 test: fmt codegen
-	go test -v -cover -coverprofile ./coverage.out ./cloud/... ./sentry/... $(TEST_ARGS)
+	go test -v -coverpkg=./sentry,./cloud/linode/client,./cloud/linode/firewall,./cloud/linode -coverprofile ./coverage.out -cover ./sentry/... ./cloud/... $(TEST_ARGS)
```

Comment:
This change was added because there was a bug introduced in go 1.23.3 which we were using https://github.com/golang/go/issues/70244. Our code coverage went down for 1.24 without this fix.